configure.ac

```diff
@@ -169,29 +171,33 @@ AS_IF([test "x$with_docs_only" = xyes],
         AC_DEFINE([UCS_MAX_LOG_LEVEL], [UCS_LOG_LEVEL_TRACE_POLL], [Highest log level])
      else
         AC_DEFINE([UCS_MAX_LOG_LEVEL], [UCS_LOG_LEVEL_INFO], [Highest log level])
-        CFLAGS="$CFLAGS -O3 -DNDEBUG"
-        CXXFLAGS="$CXXFLAGS -O3 -DNDEBUG"
+        CFLAGS="$CFLAGS -O3 -g -DNDEBUG"
+        CXXFLAGS="$CXXFLAGS -O3 -g -DNDEBUG"
      fi
 
      CHECK_UCX
      AC_MSG_RESULT([UCX support: $ucx_happy])
      if test $ucx_happy != "yes"; then
          AC_MSG_ERROR([UCX is not available])
+     else
+         tl_modules="${tl_modules}:ucp"
      fi
 
      CHECK_CUDA
      AC_MSG_RESULT([CUDA support: $cuda_happy; $CUDA_CPPFLAGS $CUDA_LDFLAGS])
+     if test $cuda_happy = "yes"; then
+         mc_modules="${mc_modules}:cuda"
+     fi
 
      CHECK_NCCL
      AC_MSG_RESULT([NCCL support: $nccl_happy])
+     if test $nccl_happy = "yes"; then
+         tl_modules="${tl_modules}:nccl"
+     fi
     ]) # Docs only
 
-
-
-
-includes="-I${UCC_TOP_SRCDIR}/src -I${UCC_TOP_BUILDDIR}/src"
 CFLAGS="$CFLAGS -std=gnu11"
-CPPFLAGS="$UCS_CPPFLAGS $CPPFLAGS $includes -Wall -Werror"
+CPPFLAGS="$CPPFLAGS $UCS_CPPFLAGS $includes"
```

Comment:
Where is "includes" variable? I cant find it. Does it compile out of src tree?

Reply:
yes, it was duplication, same set of includes was defined in BASE_CPPFLAGS
BASE_CPPFLAGS="$BASE_CPPFLAGS -I\${abs_top_srcdir}/src"
BASE_CPPFLAGS="$BASE_CPPFLAGS -I\${abs_top_builddir}"
BASE_CPPFLAGS="$BASE_CPPFLAGS -I\${abs_top_builddir}/src"